MapleServer2/Constants/SendOp.cs

```diff
@@ -2,268 +2,268 @@
+    ItemInventory = 0x0021,
+    StorageInventory = 0x0022,
+    WarehouseInventory = 0x0023,
+    FurnishingInventory = 0x0024,
+    ItemPutOn = 0x0025,
+    ItemPutOff = 0x0026,
+    ItemDrop = 0x0029,
+    ItemExtraData = 0x002A,
+    FieldAddItem = 0x002B,
+    FieldRemoveItem = 0x002C,
+    FieldPickupItem = 0x002D,
+    FieldMutateItem = 0x002E,
+    Stat = 0x002F,
+    UserBattle = 0x0030,
+    UserSkinColor = 0x0031,
+    Beauty = 0x0032,
+    AdventurerBar = 0x0033,
+    RevivalConfirm = 0x0034,
+    Revival = 0x0035,
+    UserState = 0x0037,
+    ExpUp = 0x0038,
+    LevelUp = 0x0039,
+    Meso = 0x003A,
+    Meret = 0x003B,
+    MesoToken = 0x003C,
+    SkillUse = 0x003D,
+    SkillDmg = 0x003E,
+    SkillSync = 0x003F,
+    SkillCancel = 0x0040,
+    SkillUseFailed = 0x0041,
+    SkillCooldown = 0x0043,
+    SkillResetCooldown = 0x0044,
+    SkillPoint = 0x0045,
+    StatPoint = 0x0046,
+    CharCreate = 0x0047,
+    Buff = 0x0048,
+    FieldPortal = 0x0049,
+    Job = 0x004A,
+    NpcMonologue = 0x004B,
+    NpcTalk = 0x004C,
+    RegionSkill = 0x004D,
+    FunctionCube = 0x004E,
+    Trigger = 0x004F,
+    Breakable = 0x0050,
+    Room = 0x0051,
+    Shop = 0x0052,
+    Quest = 0x0053,
+    Party = 0x0054,
+    Mail = 0x0055,
+    FieldAddNpc = 0x0056,
+    FieldRemoveNpc = 0x0057,
+    FieldDeadNpc = 0x0058,
+    NpcControl = 0x0059,
+    InteractNpc = 0x005A,
+    FieldAddPet = 0x005B,
+    FieldRemovePet = 0x005C,
+    SyncPetTamingPoint = 0x005D,
+    Tombstone = 0x005E,
+    Trophy = 0x005F,
+    DungeonWait = 0x00F2,
+    UserMoveByPortal = 0x0060,
+    ItemTitle = 0x0061,
+    MassiveEvent = 0x0062,
+    Buddy = 0x0063,
+    AdminBlock = 0x0064,
+    InteractObject = 0x0065,
+    StateConsumeEp = 0x0066,
+    StateFallDmg = 0x0067,
+    Cinematic = 0x0068,
+    Admin = 0x0069,
+    SetBuildMode = 0x006A,
+    ResponseCube = 0x006B,
+    Cubes = 0x006C,
+    Ugc = 0x006D,
+    MeretMarket = 0x006E,
+    ContentShutdown = 0x006F,
+    GuideObject = 0x0070,
+    KeyTable = 0x0071,
+    FollowNpc = 0x0072,
+    Notice = 0x0073,
+    RelocateWorld = 0x0074,
+    Liftable = 0x0075,
+    ItemFusion = 0x0076,
+    Vibrate = 0x0077,
+    HideVibrate = 0x0078,
+    ShowVibrate = 0x0079,
+    CharInfo = 0x007A,
+    ResponseRide = 0x007B,
+    RideSync = 0x007C,
+    FittingDoll = 0x007D,
+    BonusGame = 0x007E,
+    LoadUgcMap = 0x007F,
+    FieldObject = 0x0080,
+    Gem = 0x0081,
+    Taxi = 0x0082,
+    FindFields = 0x0083,
+    Trade = 0x0084,
+    InvincibleEffect = 0x0085,
+    WorldMap = 0x0086,
+    MoveEventField = 0x0087,
+    DpsStat = 0x0088,
+    DebugMode = 0x0089,
+    StoryBook = 0x008A,
+    GuideRecord = 0x008B,
+    Guild = 0x008C,
+    GroupChat = 0x008D,
+    RecallUser = 0x008E,
+    Rank = 0x008F,
+    AppendMessageCommon = 0x0090,
+    AppendMessageString = 0x0091,
+    AppendMessageKillBoss = 0x0092,
+    AppendClientLog = 0x0093,
+    AppendMessageAssistBonus = 0x0094,
+    Ah = 0x0095,
```

Comment:
rename AuctionHouse

Reply:
This isn't auction house. It's anti hack